website/docs/api/top-level.md

```diff
@@ -463,62 +463,6 @@ start decreasing across epochs.
 
  </Accordion>
 
-#### spacy.WandbLogger.v2 {#WandbLogger tag="registered function"}
```

Comment:
I'm guessing that this anchor point `#WandbLogger` is being referred to from other places of the docs too, and we'll need to update those too. Perhaps we can make sure that https://github.com/explosion/spacy-loggers#wandb or something such is a valid URL point to the latest `WandbLogger` implementation, then update the links through the docs?

We'll also need to update comments like "There are two built-in loggers" --> one built-in, with alternatives defined in https://github.com/explosion/spacy-loggers, etc.